ApplicationCode/ProjectDataModel/Rim3dOverlayInfoConfig.h

```diff
@@ -70,9 +70,9 @@ class Rim3dOverlayInfoConfig : public caf::PdmObject
         double                     sum;
         double                     weightedMean;
         const std::vector<size_t>* histogram;
-        bool isValid( double parameter ) { return parameter != HUGE_VAL && parameter != -HUGE_VAL; }
+        bool isValid( double parameter ) const { return parameter != HUGE_VAL && parameter != -HUGE_VAL; }
```

Comment:
Please move implementation to cpp

Reply:
I just added const, the implementation was already there.

Reply:
I would like to have the implementation moved as a refactoring improvement in addition to your const fix.